Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
Preferable in what way, not sure I follow. Do something with the console and check the output? I'm not sure how that could be done.